wiki/Tournaments/OWC/2013/de.md

```diff
@@ -288,155 +288,155 @@ Das Turnier kam zu einem Ende mit folgenden Platzierungen:
+| **China** ![][flag_CN] | **4** | 1 | ![][flag_SE] Schweden | [#1](https://osu.ppy.sh/community/matches/2648351) |
+| **Lettland** ![][flag_LV] | **4** | 3 | ![][flag_IT] Italien | [#1](https://osu.ppy.sh/community/matches/2648523) |
+| **Taiwan** ![][flag_TW] | **4** | 0 | ![][flag_VE] Venezuela | *siegt standardmäßig* |
+| Norwegen ![][flag_NO] | 0 | **4** | ![][flag_FR] **Frankreich** | [#1](https://osu.ppy.sh/community/matches/2651081) |
+| **Niederlande** ![][flag_NL] | **4** | 0 | ![][flag_SG] Singapur | [#1](https://osu.ppy.sh/community/matches/2649765) |
+| **Thailand** ![][flag_TH] | **4** | 0 | ![][flag_AR] Argentinien | [#1](https://osu.ppy.sh/community/matches/2652001) |
+| **Russische Föderation** ![][flag_RU] | **4** | 0 | ![][flag_FI] Finnland | [#1](https://osu.ppy.sh/community/matches/2653645) |
+| **Deutschland** ![][flag_DE] | **4** | 0 | ![][flag_BG] Bulgarien | [#1](https://osu.ppy.sh/community/matches/2655599) |
+| **Polen** ![][flag_PL] | **4** | 0 | ![][flag_EE] Estland | [#1](https://osu.ppy.sh/community/matches/2656900) |
+| **Frankreich** ![][flag_FR] | **4** | 1 | ![][flag_CL] Chile | [#1](https://osu.ppy.sh/community/matches/2660496) |
+| **Vereinigtes Königreich** ![][flag_GB] | **4** | 2 | ![][flag_CA] Kanada | [#1](https://osu.ppy.sh/community/matches/2660446) |
+| Schweden ![][flag_SE] | 2 | **4** | ![][flag_AT] **Österreich** | [#1](https://osu.ppy.sh/community/matches/2661584) |
+| **Südkorea** ![][flag_KR] | **4** | 2 | ![][flag_CL] Chile | [#1](https://osu.ppy.sh/community/matches/2662493) |
+| Australien ![][flag_AU] | 1 | **4** | ![][flag_BR] **Brasilien** | [#1](https://osu.ppy.sh/community/matches/2767400) |
 
 ## Regelsätze
 
 ### Turnierregeln
 
 1. Das osu! World Cup ist ein länderbezogenes 4 gegen 4 Team Turnier.
-2. Die Beatmaps werden vom Map Wähler für jede Runde im Vorhinein am Sonntag angekündigt, bevor die eigentliche Partie stattfindet. Nur diese werden dann während den eigentlichen Partien zur Verfügung stehen.
+2. Die Beatmaps werden von den Beatmap-Wählern für jede Runde im Vorhinein am Sonntag angekündigt, bevor die eigentliche Partie stattfindet. Nur diese werden dann während den eigentlichen Partien zur Verfügung stehen.
    - Eine Beatmap wird als Tiebreaker-Map mit angegeben. Diese Beatmap wird nur beim größtmöglichem Punktegleichstand gespielt.
-   - Es wird auch einen [Hidden](/wiki/Game_modifier/Hidden), [HardRock](/wiki/Game_modifier/Hard_Rock), [DoubleTime](/wiki/Game_modifier/Double_Time) und FreeMod Stappel geben.
-3. Die Zeiten der Partien werden vom Turnier Management festgelegt (siehe unten).
-4. Wenn kein Schiedrichter oder Personal vorhanden sein sollte, wird die Partie verschoben.
-5. Die Ergebnisse von wiederbelebten Spielern wird zum Teamergebnis mit angerechnet.
+   - Es wird auch einen [Hidden](/wiki/Game_modifier/Hidden), [HardRock](/wiki/Game_modifier/Hard_Rock), [DoubleTime](/wiki/Game_modifier/Double_Time) und FreeMod Stapel geben.
```

Comment:
```suggestion
   - Es wird auch einen [Hidden](/wiki/Game_modifier/Hidden)-, [HardRock](/wiki/Game_modifier/Hard_Rock)-, [DoubleTime](/wiki/Game_modifier/Double_Time)- und FreeMod-Stapel geben.
```
Kinda disliking this a bit. An alternative might be `Es wird auch einen Stapel für Hidden, HardRock, DoubleTime und FreeMod geben`.